spec/Route_tests/posts_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
```

Comment:
- [ ] The `users_spec.rb` and `posts_spec.rb` file should be in `spec/requests/` directory. You can change the directory name "Route_tests" to  "requests". :+1: